src/crawlee/storages/_dataset.py

```diff
@@ -77,6 +79,71 @@ class ExportToKwargs(TypedDict):
     """Name of the key-value store to save the exported file."""
 
 
+class ExportDataJsonKwargs(TypedDict):
+    """Keyword arguments for dataset's `export_data_json` method.
+
+    Args:
+        skipkeys: If True (default: False), then dict keys that are not of a basic type (str, int, float, bool, None)
+            will be skipped instead of raising a TypeError.
+        ensure_ascii: Whether non-ASCII characters should be escaped in the output JSON string.
+        check_circular: If False (default: True), then the circular reference check for container types will be skipped
+            and a circular reference will result in a RecursionError (or worse).
+        allow_nan: If False (default: True), then it will be a ValueError to serialize out of range float values
+            (nan, inf, -inf) in strict compliance of the JSON specification. If allow_nan is true, their JavaScript
+            equivalents (NaN, Infinity, -Infinity) will be used.
+        cls: Allows using a custom JSON encoder
+        indent: Specifies the number of spaces to use for indentation in the pretty-printed JSON output.
+            separators: a (item_separator, key_separator) tuple. The default is (', ', ': ') if indent is None
+            and (',', ': ') otherwise.
+        default: A function that gets called for objects that can't otherwise be serialized.
+            It should return a JSON encodable version of the object or raise a TypeError.
+        sort_keys: Specifies whether the output JSON object should have its keys sorted alphabetically.
+    """
+
+    skipkeys: NotRequired[bool]
+    ensure_ascii: NotRequired[bool]
+    check_circular: NotRequired[bool]
+    allow_nan: NotRequired[bool]
+    cls: NotRequired[type[json.JSONEncoder]]
+    indent: NotRequired[int]
+    separators: NotRequired[tuple[str, str]]
+    default: NotRequired[Callable]
+    sort_keys: NotRequired[bool]
+
+
+class ExportDataCsvKwargs(TypedDict):
+    r"""Keyword arguments for dataset's `export_data_csv` method.
```

Comment:
Is `r"""` intented?

Reply:
Yes, because of backslashes in the docstring. It doesn't seem to break docs build :smile: